app/src/main/java/edu/agh/susgame/front/ui/util/ZoomState.kt

```diff
@@ -31,8 +32,13 @@ class ZoomState(
         scale.floatValue = max(min(scale.floatValue * zoom, maxZoomIn), maxZoomOut)
     }
 
-    fun move(offset: Offset) {
-        translationX.floatValue += offset.x
-        translationY.floatValue += offset.y
+
+    fun move(offset: Offset) { // it doesn't work as it should, right now i don't have any idea how to implement it :(
```

Comment:
Hmm faktycznie trochę dziwnie się to zachowuje, ale w miarę działa. Proponuję:
- Dodaj `TODO GAME-47` do komentarza
- Zaktualizuj tamten ticket — nowy tytuł to np. `Napraw blokadę wychodzenia poza mapę`, a w opisie opisz krótko co jest teraz źle